feature-libs/quote/occ/adapters/occ-quote.adapter.spec.ts

```diff
@@ -322,6 +337,59 @@ describe(`OccQuoteAdapter`, () => {
     );
   });
 
+  it('downloadAttachment should download proposal document based on provided quoteCode and attachmentId', (done) => {
```

Comment:
@suprishi, could you group all your tests into:
describe('downloadAttachment', () => {
...
});

then you don't need to add `downloadAttachment` to every your test ;)